corehq/apps/users/views/mobile/users.py

```diff
@@ -311,7 +312,9 @@ def page_context(self):
 
     @property
     def user_role_choices(self):
-        return [('none', _('(none)'))] + self.editable_role_choices
+        role_choices = self.editable_role_choices
+        default_role = UserRole.commcare_user_default(self.domain)
+        return [(default_role.get_qualified_id(), default_role.name or _('Default Role'))] + role_choices
```

Comment:
What's the ` or _('Default Role')` bit for?  Like if the user clears it or something?

Reply:
I was just copying what was there before but it's probably not necessary since you can't save a role with no name. I'll remove this.